src/main/java/io/stargate/sgv2/jsonapi/api/model/command/impl/AlterTableCommand.java

```diff
@@ -7,10 +7,9 @@
 import org.eclipse.microprofile.openapi.annotations.media.Schema;
 
 @Schema(description = "Command that alters the column definition in a table.")
-@JsonTypeName("alterTable")
+@JsonTypeName(CommandName.Names.ALTER_TABLE_NAME)
```

Comment:
After the last merge I think we can set this as `CommandName.ALTER_TABLE.getApiName()`

Reply:
Unfortunately no -- annotation values must be actual constants, like String constants; cannot use `Constant.method()` :-(
(I tried that, won't compile)